UITests/TransferMethod/ListTransferMethodTests.swift

```diff
@@ -29,17 +28,7 @@ class ListTransferMethodTests: XCTestCase {
     }
```

Comment:
since it is public repo, thus we should replace the mock token instead of the real one

Reply:
replaced